.changeset/four-squids-change.md

```diff
@@ -0,0 +1,7 @@
+---
+'@ember-headless-form/changeset': major
```

Comment:
Out of curiosity I tagged this as a major release, to see what that will lead to. Can change if that doesn't make any sense...

Reply:
it does make sense! that'll start us at 1.0.0-beta (since we're in beta mode)